gapis/gfxapi/vulkan/vulkan.api

```diff
@@ -4607,6 +4607,11 @@ cmd VkResult vkResetDescriptorPool(
     VkDevice                   device,
     VkDescriptorPool           descriptorPool,
     VkDescriptorPoolResetFlags flags) {
+  sets := DescriptorPools[descriptorPool].DescriptorSets
```

Comment:
I think so, although I would look at the generated golang code.
I have a feeling that in golang we don't actually copy the map, but reference it.